src/main/webapp/resources/js/components/sequence-files/SequenceFileHeader.jsx

```diff
@@ -14,6 +14,8 @@ import { SequenceFileHeaderOwner } from "./SequenceFileHeaderOwner";
  * @param type The type of file object (sequencingobject or genomeassembly)
  * @function remove files from sample function
  * @param displayConcatenationCheckbox Whether to display checkbox or not
+ * @function set default sequencing object for sample
+ * @param autoDefaultFirstPair the first pair in the list of pairs (if displaying paired end files)
```

Comment:
Same as previous, don't us `@function` here

Reply:
Updated in a23c555bf4b038f2ee0e43742e27ff837b87eb78

Reply:
Thanks